charts/nginx-gateway-fabric/README.md

```diff
@@ -246,16 +246,18 @@ kubectl delete -f https://github.com/kubernetes-sigs/gateway-api/releases/downlo
 | `nginxGateway.extraVolumeMounts`                        | Extra `volumeMounts` for the nginxGateway container.                                                                                                                                                     | {}                                                                                                              |
 | `nginxGateway.gatewayClassName`                         | The name of the GatewayClass for the NGINX Gateway Fabric deployment.                                                                                                                                    | nginx                                                                                                           |
+| `nginxGateway.gatewayClassAnnotat`                      | Set of custom annotations for GatewayClass objects.                                                                                                                                                      | {}                                                                                                              |
 | `nginxGateway.gatewayControllerName`                    | The name of the Gateway controller. The controller name must be of the form: DOMAIN/PATH. The controller's domain is gateway.nginx.org.                                                                  | gateway.nginx.org/nginx-gateway-controller                                                                      |
 | `nginxGateway.kind`                                     | The kind of the NGINX Gateway Fabric installation - currently, only Deployment is supported.                                                                                                             | deployment                                                                                                      |
 | `nginxGateway.config`                                   | The dynamic configuration for the control plane that is contained in the NginxGateway resource                                                                                                           | [See nginxGateway.config section](values.yaml)                                                                  |
+| `nginxGateway.Annotations`                              | Set of custom annotations for NginxGateway objects.                                                                                                                                                      | {}                                                                                                              |
```

Comment:
shouldn't this be `configAnnotations` as thats the described field.

Reply:
also fixed